### ISSUE TITLE
customStatement missing call to doWhenOpened

### ISSUE THREAD

**Issue by Mike278** [simolus3/moor#199]
Otherwise trying to call `customStatement` immediately after instantiating the database or inside a transaction throws an error:
```dart
final db = MyDatabase(VmDatabase.memory(logStatements: true));
//await db.select(db.someTable).get(); // uncomment to fix select 1=2
await db.customStatement("select 1=2");
await db.transaction(() async {
  await db.customStatement("select 1=1");
});
```
```
package:moor_ffi/src/vm_database.dart 64:17  _VmDelegate._runWithArgs
NoSuchMethodError: The method 'execute' was called on null.
Receiver: null
Tried calling: execute("select 1 = 2")

//...

package:moor/src/runtime/executor/helpers/engines.dart 71:19  _ExecutorWithQueryDelegate.runCustom.<fn>
package:synchronized/src/basic_lock.dart 31:26                BasicLock.synchronized
NoSuchMethodError: The method 'runCustom' was called on null.
Receiver: null
Tried calling: runCustom("select 1 = 1", _ImmutableList len:0)
```

**Comment by simolus3** [comment]
Thanks for the report, should be fixed on `develop` now. I'll cherry-pick that commit to master and release a hotfix to pub later.

**Comment by simolus3** [comment]
Fixed in 2.0.1+1 of `moor`.
